src/views/virtualmachines/details/tabs/configuration/metadata/MetadataTab.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import { V1VirtualMachine, V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { AnnotationsModal } from '@kubevirt-utils/components/AnnotationsModal/AnnotationsModal';
+import { LabelsModal } from '@kubevirt-utils/components/LabelsModal/LabelsModal';
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import VirtualMachineDescriptionItem from '@kubevirt-utils/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { DescriptionList, Grid, PageSection, Title } from '@patternfly/react-core';
+
+import { updateAnnotation, updateLabels } from '../details/utils/utils';
+
+import MetadataTabAnnotations from './components/MetadataTabAnnotations/MetadataTabAnnotations';
+import MetadataTabLabels from './components/MetadataTabLabels/MetadataTabLabels';
+
+type MetadataTabProps = RouteComponentProps<{
+  name: string;
+  ns: string;
+}> & {
+  obj?: V1VirtualMachine;
+  vmi?: V1VirtualMachineInstance;
+};
+
+const MetadataTab: FC<MetadataTabProps> = ({ obj: vm }) => {
+  const { t } = useKubevirtTranslation();
+  const { createModal } = useModal();
+
+  return (
+    <PageSection>
+      <Title headingLevel="h2">{t('Metadata')}</Title>
+      <Grid span={6}>
+        <DescriptionList>
+          <VirtualMachineDescriptionItem
+            bodyContent={t(
+              'Map of string keys and values that can be used to organize and categorize (scope and select) objects. May match selectors of replication controllers and services. ',
```

Comment:
Do you know if the spaces at the end of bodyContent strings are necessary?

Reply:
yes the space is needed i just checked